homeassistant/components/doods/image_processing.py

```diff
@@ -31,6 +32,7 @@
 CONF_DETECTOR = "detector"
 CONF_LABELS = "labels"
 CONF_AREA = "area"
+CONF_CONTAINS = "contains"
```

Comment:
I think that the name is not descriptive enough. It is used in a way that it should only trigger if the object is fully contained within the area. 

Can we change the word to better describe that this is an all or nothing boolean. Because if an area contains a person for 50%, it doesn't trigger.

Not sure about the right word here, maybe something like `fully_contains` ?

Reply:
How about "covers"? in set theory, when A covers B it means B is fully contained in A.